frontend/src/utils/relayer.ts

```diff
@@ -2,20 +2,25 @@
  * @notice Class for managing relayed withdrawal transactions
  */
 
-import { JsonRpcProvider } from 'src/utils/ethers';
+import { JsonRpcProvider, BigNumber, parseUnits } from 'src/utils/ethers';
 import {
   ConfirmedITXStatusResponse,
   FeeEstimateResponse,
   ITXStatusResponse,
   Provider,
-  TokenInfo,
+  TokenInfoWithMinSendAmount,
   RelayResponse,
   TokenListResponse,
   WithdrawalInputs,
 } from 'components/models';
 
 export class ITXRelayer {
-  constructor(readonly baseUrl: string, readonly tokens: TokenInfo[], readonly chainId: number) {}
+  constructor(
+    readonly baseUrl: string,
+    readonly tokens: TokenInfoWithMinSendAmount[],
+    readonly chainId: number,
+    readonly nativeTokenMinSendAmount: BigNumber | undefined,
```

Comment:
Same comment about not needing to special-case the native token and just include it in the tokens array